src/styles/variables.scss

```diff
@@ -0,0 +1,7 @@
+$black: #060b35;
```

Comment:
it is a good practice to name variables with color values with name `$color-` at start, please fix it for all variables to be e.g:

```
$color-black:
$color-white:

etc...
```